### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

Data correction for items 18107, 18108, 18124, 18151, 21038, 24166, 24167

https://divine-pride.net/database/item/18107
https://divine-pride.net/database/item/18108
https://divine-pride.net/database/item/18124
``````
remove bonus bUnbreakableWeapon
``````

https://divine-pride.net/database/item/18151
``````
insert skill level  DC_DANCINGLESSON to bonus bLongAtkRate 
``````

https://divine-pride.net/database/item/21038
https://divine-pride.net/database/item/24166
https://divine-pride.net/database/item/24167
``````
update script
``````

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -92875,18 +92869,12 @@ Body:
...
-         bonus bBaseAtk,.@r/2;
-      }
-      if (.@r >= 3) {
-         bonus2 bSkillAtk,"RK_SONICWAVE",.@r/3*10;
-      }
-      if (.@r >= 9) {
-         bonus5 bAutoSpell,"RK_SONICWAVE",2,.@r/3*10,BF_SHORT|BF_WEAPON,1;
-      }
-      if (.@r >= 11) {
+      bonus bBaseAtk,.@r/2;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:92872]:
```suggestion
      bonus bBaseAtk,10*(.@r/2);
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -97868,9 +97856,7 @@ Body:
     EquipLevelMin: 1
     Refineable: true
     Script: |
-      .@r = getrefine();
-      bonus2 bIgnoreDefRaceRate,RC_All,5+(.@r/2);
-      bonus2 bIgnoreDefRaceRate,RC_Player_Human,-5-(.@r/2);
+      bonus2 bIgnoreMdefClassRate,Class_Normal,5+(getrefine()/2);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:97859]: From Aegis
```suggestion
      .@r = getrefine();
      bonus bMaxHP,.@r*10;
      bonus2 bIgnoreDefClassRate,Class_Normal,5+(.@r/2);
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -97880,9 +97866,7 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:97869]: From Aegis
```suggestion
      .@r = getrefine();
      bonus bMaxHP,.@r*10;
      bonus2 bIgnoreDefClassRate,Class_Normal,5+(.@r/2);
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:92872]: Thanks

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:97859]: Thanks

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:97869]: Thanks

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**